changelog/unreleased/issue-3312

```diff
@@ -1,6 +1,6 @@
 Enhancement: Add auto-completion file for fish
 
-The version of cobra had been increased to v1.1.3, and `generate` command has added 
-auto-completion file for fish.
+We added support for fish auto completion for the `generate` command by
+upgrading the Cobra library.
```

Comment:
No biggie but it's not really needed to explain how a fix was actually implemented (in this case "by upgrading the Cobra library", that's besides the point :D